home/templates/home/question_detail.html

```diff
@@ -44,7 +44,7 @@ <h3> {{ question.title }} </h3>
 	<div class="answer-display mt-1 col-md-10">
 	<div class="row">
 		<div class="asnwer-conent col-md-10">
-			<p>{{ answer.content }}</p>
+			<p>{{ answer.content | safe }}</p>
```

Comment:
This may or may not be a security issue - does the RichTextField class perfom any HTML sanitation to ensure there are not things like `<script>` tags?

Reply:
@ifireball fixed it in #131 , wait for merge and then i will use the same solution here.

Reply:
fixed